Buildings/Fluid/AirFilters/BaseClasses/Characteristics/filtrationEfficiencyParameters.mo

```diff
@@ -0,0 +1,27 @@
+within Buildings.Fluid.AirFilters.BaseClasses.Characteristics;
+record filtrationEfficiencyParameters
+  "Record for filtration efficiency vs. relative mass of the contaminant captured by the filter"
+  extends Modelica.Icons.Record;
+  parameter Real rat[:,:](each max=1, each min=0)
+    "Relative mass of the contaminant captured by the filter";
+  parameter Real eps[:,:](each max=1, each min=0)
+    "Filtration efficiency";
+
+  annotation (Documentation(info="<html>
+<p>
+Data record that describes the relative mass of the contaminant captured by the filter <code>rat</code> versus
+the filtration efficiency <code>eps</code>.
```

Comment:
`Data record that describes the relative mass of the contaminant <code>rat</code> captured by the filter when it has the filtration efficiency specified as <code>eps</code>.`

Reply:
Update it as suggested